roles/jenkins_slave/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+- name: install rpm packages
+  yum:
+    name: "{{ jslave_rpm_deps | join(',') }}"
+    state: present
+
+- name: install pip packages
```

Comment:
Agreed, this is less than ideal, but it's the best we can do for now and is part of the port from the previous version of this software.  If you have an idea to make this better we can discuss it offline and turn it into an actionable ticket.